src/shader/billboard.wgsl

```diff
@@ -45,11 +45,11 @@ fn vertex(vertex: Vertex) -> VertexOutput {
     let vertex_position = vec4<f32>(-vertex.position.x, vertex.position.y, vertex.position.z, 1.0);
     let position = view.view_proj * billboard.model * vertex_position;
 #else
-    let camera_right = normalize(vec3<f32>(view.view_proj.x.x, view.view_proj.y.x, view.view_proj.z.x));
+    let camera_right = normalize(vec3<f32>(view.view_proj[0][0], view.view_proj[1][0], view.view_proj[2][0]));
```

Comment:
I was getting invalid field accessors. I wonder if this was a change in naga, but I did not investigate.
```
error: invalid field accessor `x`
   ┌─ src/shader/billboard.wgsl:48:59
   │
48 │     let camera_right = normalize(vec3<f32>(view.view_proj.x.x, view.view_proj.y.x, view.view_proj.z.x));
   │                                                           ^ invalid accessor
   │
   = invalid field accessor `x`
```